README.md

```diff
@@ -1,13 +1,29 @@
 # Magento 2 PCI DSS 4.0 Compatibility
 
-A Magento 2 module to bring it in-line with the [PCI DSS 4.0 requirements](https://east.pcisecuritystandards.org/document_library?category=pcidss&document=pci_dss), with changes including:
-* Automatic disabling of admin accounts with 90 days of inactivity.
-   * Functionality added via new cron job that runs once per day.
-* Restriction of admin session timeout to be no more than 15 minutes.
-* Restriction of admin lockout functionality:
-   * No more than 10 attempts before lockout.
-   * Lockout duration of no less than 30 minutes.
-* Enforce than admin passwords must contain at least 12 characters.
+A Magento 2 module to bring it in-line with the [PCI DSS 4.0 requirements](https://east.pcisecuritystandards.org/document_library?category=pcidss&document=pci_dss), with changes covering the following requirements:
```

Comment:
> A Magento 2 module to bring it in-line with the

I don't know why, but "it" bothers me. Should we specfiy "the CMS" or "the admin area" or something slightly more detailed? Or "A module to bring Magento 2 in-line with..." maybe

Reply:
No, that's fair - I'll update it.